Yko/app open (#2)
qlik-product-success/qs-admin-playbook: docs/performance/app_open_performance.md

The engine goes through many stages when a user wants to attach to an app, and some of those stages will depend on whether or not the app is already loaded into memory. The following progress report shows a typical case of a "cold open", where the app has not already been loaded into memory:

> ![Progress_report_cold_open.png](images/Progress_report_cold_open_highlighted.png)

Reviewer: Suggesting to have image number to easier be able to explain the comparison later.

Example (or something similar):
```suggestion
![Progress_report_cold_open.png](images/Progress_report_cold_open_highlighted.png)
(image 1.1)
```

Author: Done

Notice how much of the time (about 48 seconds in total) is spent on loading fields and tables. This is the engine reading the QVF, unpacking the data, and building the data structures required to perform calculations on the app. Compare that to the following progress report for the exact same app:

> ![Progress_report_warm_open.png](images/Progress_report_warm_open.png)

Reviewer: Suggesting to have image number to easier be able to explain the comparison later.

Example (or something similar):
```suggestion
![Progress_report_warm_open.png](images/Progress_report_warm_open.png)
(image 1.2)
```

Author: Done

> Here we can see that the loading of fields and tables are not present. In fact, if we compare line four of the two progress reports, we can see that the first one states "Loading from storage" while the second one reads "Attached to loaded document". This means that the app was alreay loaded into memory when the second connection was performed.

Reviewer: Would be nice to have image number or something that you can refer to. I think it would make it easier to point to different images when you explain this.

Example:
```suggestion
Here we can see that the loading of fields and tables are not present. In fact, if we compare line four of the two progress reports, we can see that the image 1.1 states "Loading from storage" while the image 1.2 reads "Attached to loaded document". This means that the app was alreay loaded into memory when the second connection was performed.
```

Author: Done.

Here we can see that the loading of fields and tables are not present. In fact, if we compare line four of the two progress reports, we can see that the first one states "Loading from storage" while the second one reads "Attached to loaded document". This means that the app was alreay loaded into memory when the second connection was performed.

> If users complain that apps are sometimes slow to open, then this is the first thing to look for. If it is clear that the loading of fields and tables is the main cause of slow app open times, then implementing some form of pre-caching technique in your environment might be a good option.

Reviewer: Can we refer to another whitepaper for pre-caching or pre-loading of apps?
Or perhaps refer it to qs-admin-playbook page in future

Author: I do have a cross reference to the "Cache warming" section of the playbook in there. Is that sufficient, or are you referring to something else?

Reviewer: The cross reference is good enough. Thanks
